src/shared/views/BuildingFootprint/Validator.js

```diff
@@ -2,145 +2,218 @@
 import * as React from 'react';
 import { compose } from 'redux';
 import { connect } from 'react-redux';
+import pako from 'pako';
+import base64 from 'base-64';
 import { firebaseConnect, isEmpty, isLoaded } from 'react-redux-firebase';
 import { StyleSheet, View } from 'react-native';
 import Button from 'apsl-react-native-button';
 import FootprintDisplay from './FootprintDisplay';
 import LoadingIcon from '../LoadingIcon';
-import { COLOR_GREEN, COLOR_RED, COLOR_YELLOW } from '../../constants';
+import { COLOR_DEEP_BLUE, COLOR_WHITE } from '../../constants';
 
 import type {
     BuildingFootprintGroupType,
-    ProjectType,
+    SingleImageryProjectType,
     BuildingFootprintTaskType,
 } from '../../flow-types';
 
 const styles = StyleSheet.create({
-    button: {
-        borderRadius: 0,
+    container: {
+        alignItems: 'center',
+        flex: 1,
+        flexDirection: 'column',
+        justifyContent: 'flex-start',
+    },
+    sideBySideButtons: {
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        height: 105,
+    },
+    bigSquareButton: {
+        backgroundColor: COLOR_WHITE,
+        borderRadius: 20,
+        height: 100,
+        width: '47%',
+        marginBottom: 5,
+        marginLeft: 5,
+        marginRight: 5,
+        marginTop: 5,
+    },
+    bigSquareButtonText: {
+        color: COLOR_DEEP_BLUE,
+        fontWeight: 'bold',
+    },
+    longNarrowButton: {
+        borderColor: COLOR_WHITE,
+        borderRadius: 20,
+        borderWidth: 2,
+        color: COLOR_WHITE,
         height: 55,
         marginBottom: 0,
+        marginLeft: 5,
+        marginRight: 5,
         marginTop: 5,
     },
+    longNarrowButtonText: {
+        color: COLOR_WHITE,
+        fontWeight: 'bold',
+    },
 });
 
-const FOOTPRINT_CORRECT = 1;
-const FOOTPRINT_NEEDS_ADJUSTMENT = 2;
-const FOOTPRINT_NO_BUILDING = 3;
+const WSF_NO_BUILDING = 0;
```

Comment:
I think you could have this more generic:

FOOTPRINT_NO = 0
FOOTPRINT_YES = 1
FOOTPRINT_MAYBE = 2
FOOTPRINT_BAD = 3

For footprint we will (for now) always have a yes-no question. So the above will be valid for all projects, and no need to do it WSF specific here.

Reply:
fixed!